moduleroot/Rakefile.erb

```diff
@@ -66,8 +66,8 @@ end
 <% checks.each do |check| -%>
 PuppetLint.configuration.send('disable_<%= check %>')
 <% end -%>
-<% if @configs['linter_fail_on_warnings'] -%>
-PuppetLint.configuration.fail_on_warnings = true
+<% if ! @configs['linter_fail_on_warnings'].nil? -%>
+PuppetLint.configuration.fail_on_warnings = <%= @configs['linter_fail_on_warnings'] %>
```

Comment:
Hey @nabertrand! Apologies for the delay,
should we set a default value of true for`linter_fail_on_warnings` in `config_defaults.yml`?

Reply:
@nabertrand apologies. Missed this notification completely. 
Yeah okay, this makes sense. This is only added to the template if the user has a value set in their .sync.yml so my comments on this possibly being a breaking change don't actually apply, it will take their predetermined value.

Happy to merge!